scripts/fetch_dependencies.sh

```diff
@@ -1,30 +1,52 @@
 #!/usr/bin/env bash -e
 
-# usage: ./scripts/fetch_dependencies.sh --integration=[integration]
+# usage: ./scripts/fetch_dependencies.sh --integration=[integration] --version=[version]
 #
-# Available options: "cocoapods", "carthage"
+# Available options: "cocoapods", "spm"
 
 # parse arguments
 INTEGRATION=""
+VERSION_NUMBER=""
 
 while [[ $# -gt 0 ]]
 do
   case $1 in
     -i|--integration)
     INTEGRATION="$2"
     echo "> Following integration option was provided to the script: $INTEGRATION"
+    shift 2;; # past argument
+    -v|--version)
+    VERSION_NUMBER="$2"
+    echo "> Following version number was provided to the script: $VERSION_NUMBER"
     shift # past argument
     continue;;
     *)
     break;;
   esac
 done
 
+function updatePods {
+  pod update --repo-update
+}
+
+function updateSPM {
+  sed -i '' -e "s|version =.*|version = '$VERSION_NUMBER';|g" StreamChatIntegration-SPM.xcodeproj/project.pbxproj
```

Comment:
Can you add a comment on what this does? I assume it changes the version to which we are pointing, right?